### PR TITLE
DF 935 central config feign properties

### PR DESCRIPTION
**Pull request by emina-str**

### CODE REVIEW

**Comment by fabiangotzen** [comment]
Looks good, we should however point this to develop since master goes to prod. (I don't have admin rights on this repo but we should maybe safeguard this somehow)

File: application-development.yml [hunk 1]
```diff
@@ -49,7 +49,7 @@ jhipster:
     security:
         authentication:
             jwt:
-                secret: '{cipher}4b8f87636c1795d3847d71e3551f4c90458cc8893456a021ae616665b985561b35c0f3bc90b8cb050c342a584834a6a9095298b0645ac5d1252cfe4c63cc5a00'
+                secret: '{cipher}56b0fa62119cbacb70affae2feaa868aee6360d66b992838f31b247f6c50ba9f8536a7b1b3cc38ad5acc3f9440bdf71c019fb0f1d38ac1726bf8db6d6dd734bdd5fc80c4c5620c8115e2ea01a6d2670836b2dc498d5b145c4bec09a156cbe3d6cb61affbb429b77db883fbcec34188b8'
```
Comment (birom) [inline application-development.yml]: Why did we change this?

Comment (emina-str) [reply application-development.yml]: @birom  We don't. It was probably some left over from resolving conflicts with develop (as the branch was created from master). Now, it should be ok.

Comment (birom) [reply application-development.yml]: Thx, could you please check, if the 

```
jhipster:
    security:
        authentication:

        client-authorization:
            access-token-uri: http://${infrastructure.gateway.hostname}:8444/api/authenticate
            client-id: '{cipher}46b903df215cc633fb2d095b04c56146573166bf241e01f977faf354f5a458a4'
            client-secret: '{cipher}df476d1d9650d9b06e9aac1d6351e19051bd9a73926b831b6b955c4c55f486ed'
```

part still in use.

**Pull request merged**